author/ogc/topics/requirements.adoc

```diff
@@ -524,13 +524,13 @@ attributes:
 * `subject`. Associated Requirement class. May be encoded as a cross-reference
 or as plain text. Rendered as _Requirement Class_.
 
-* Name. Specified as the block caption.
+* `name`. Specified as the block caption.
 
 * `inherit` (optional). Dependencies of the conformance class. Accepts multiple
 semicolon-delimited values, which each could be a cross-reference to another
 conformance class or plain text.
 
-* Nesting (optional). Conformance tests contained in a conformance class are
+* `nesting` (optional). Conformance tests contained in a conformance class are
```

Comment:
I intentionally did not use `name` because it is not the name of the attribute. I can see where this can be confusing though. Maybe we move "Name" and "Nesting" to a separate paragraph?